src/components/Table/Table.jsx

```diff
@@ -37,13 +40,22 @@ export function Table({
     order: null,
   },
   tableClassName,
+  tableDescription,
   tableWidth = { min: '700px' },
+  withColumnChooser = false,
   withFocusableRows = false,
   withSelectableRows = false,
   withTallRows = false,
   ...rest
 }) {
-  const [state, actions] = useTable(data, maxRowsToDisplay)
+  const defaultColumns = columns.map(col => {
+    if (col.show == null) {
```

Comment:
Is the `==` intentional or can this be `===`?

Reply:
intentional, `== null` is a way to catch `null` or `undefined`, little side note, you'll notice eslint doesn't complain about it, whereas it will in any other case.

Reply:
TIL!